images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -559,3 +559,28 @@ function Invoke-SBWithRetry {
         }
     }
 }
+
+function Get-GitHubPackageDownloadUrl {
+    param (
+        [string]$RepoOwner,
+        [string]$RepoName,
+        [string]$BinaryName,
+        [string]$Version,
+        [string]$UrlFilter
+    )
+
+    if ($Version -eq "latest") { 
+        $Version = "*" 
+    }
+    $json = Invoke-RestMethod -Uri "https://api.github.com/repos/${RepoOwner}/${RepoName}/releases?per_page=100"
+    $versionToDownload = ($json.Where{ -not $_.prerelease }.tag_name |
```

Comment:
Maybe create a variable `IsPrerelease` and set it to false by default in case we need to install some prerelease version one day?